samples/matter/bridge/src/bridge_manager.cpp

```diff
@@ -70,11 +90,14 @@ CHIP_ERROR BridgeManager::AddBridgedDevice(BridgedDevice::DeviceType bridgedDevi
 		return CHIP_ERROR_NO_MEMORY;
 	}
 
+	dataProvider->Init();
 	CHIP_ERROR err = AddDevice(device.get());
 
-	/* In case adding succedeed, release the ownership. */
+	/* In case adding succedeed map the objects relationship and release the ownership. */
 	if (err == CHIP_NO_ERROR) {
+		mDevicesMap[device.get()] = dataProvider.get();
```

Comment:
Yes, I know, this is a temporary solution. We are going to allocate providers and devices independently, so it will be possible to e.g map three devices to one provider. I did it in a messy way, as @markaj-nordic started to work in parallel on implementing new factory module being responsible for allocating the objects.

Reply:
@kkasperczyk-no also, do we aim to support one bridged device to many providers relation (or the other way around)? If so, please note that std::map does not allow for key duplicates.

Reply:
We may discuss it, but I didn't plan it. We may represent one Bluetooth LE device (provider), as few Matter endpoints, however I think representing few Bluetooth LE devices, as an one endpoint doesn't make much sense. If we would like to control multiple Bluetooth LE devices, as we would do it with a single device, we should use Matter groups for multiple endpoints.